### DISCUSSION TITLE
The characteristic fire diameter (D*)

### DISCUSSION THREAD

**Discussion by nevertrustenvironment** [firemodels/fds#12873] [Q&A]
Hello everyone!.
I found the equation in the FDS user guide, but I was unable to calculate the required grid size for the model.

Q= heat release rate? or HRRPUA?

dx represents the cell size, but how can we choose the value of dx?

Thank you. 

![equation](https://github.com/firemodels/fds/assets/68212891/2c670a20-bd1e-4f1a-9ee9-7b3ea926c3cf)

**Comment by Er9y714** [comment]
Q is heat release rate (kW)
dx is nominal size of a mesh cell where the burner and flame zone are located.

For comparison, you can check Validation Guide document and the range of the D*/dx of the simulated cases. 

![image](https://github.com/firemodels/fds/assets/26166281/a60980c9-414f-4f45-8434-f0ca49e59f57)

**Reply by nevertrustenvironment** [reply]
Thank you for the answer!. Suppose my mesh size for the fire is 0.5 m  x 0.5 m. So dx is 0.5 or (0.5)^2? Length of the fire burner is 2m x 2 m.

**Reply by drjfloyd** [reply]
dx is a length or in your example 0.5 m.

**Comment by drjfloyd** [comment]
6.3.6 Mesh Resolution

A common question asked by new FDS users is, “What should my grid spacing be?” The answer is not easy
because it depends considerably on what you are trying to accomplish. In general, you should build an FDS
input file using a relatively coarse mesh, and then gradually refine the mesh until you do not see appreciable
differences in your results. This is referred to as a mesh sensitivity study.

**Reply by nevertrustenvironment** [reply]
Thank you for the reply and the good suggestion.  I read some articles  that mention the initial mesh size for the simulation was considered from this equation, so I want to check it for my study. Thank you.
